scripts/config.py

```diff
@@ -191,6 +191,7 @@ def realfull_adapter(_name, active, section):
     #pylint: disable=line-too-long
     'MBEDTLS_AES_ONLY_128_BIT_KEY_LENGTH', # interacts with CTR_DRBG_128_BIT_KEY
     'MBEDTLS_AES_USE_HARDWARE_ONLY', # hardware dependency
+    'MBEDTLS_BLOCK_CIPHER_NO_DECRYPT', # incompatible with DES/CBC/XTS/NIST_KW
```

Comment:
That's a correct justification, but IMO just "negative option" was also a correct justification :)